ModelFiles/dependencies.txt

```diff
@@ -1,7 +1,7 @@
-MATLAB	9.1.0.441655 (R2016b)
+MATLAB	9.3.0.713579 (R2017b)
 libSBML	5.17.0
 RAVEN_toolbox	2.0.4
-COBRA_toolbox	commit 74dc79a
+COBRA_toolbox	unknown
```

Comment:
@hongzhonglu this points to the fact your COBRA is not properly installed. Did you **clone** the repository + initialized the toolbox?

Reply:
@BenjaSanchez  I have clone the repo and installed the latest cobra

Reply:
@BenjaSanchez It may due to I don't put the MATLAB-git-master into the path?

Reply:
yes, probably. Please ensure you have all of the [developing requirements](https://github.com/SysBioChalmers/yeast-GEM#required-software---contributor) when submitting PRs.

Reply:
OK. will be careful in next time.